source-browser.html

```diff
@@ -132,7 +156,7 @@ <h1 id="loading" class="m1 center display-none">⌛</h1>
           $('#topics').append(
             $('<div class="topic flex-auto">').append(
               $('<div>').append(
-                $('<h2 class="inline-block m0">').text(name),
+                $('<h2 class="inline-block m0" id="topic" style="padding-top:75px; margin-top:-75px;">').text(name),
```

Comment:
Don't use inline styling. Instead, style the associated class. Furthermore, this creates multiple elements with the same id, i.e. `topic`. This is illegal HTML. Instead, something like `topic-languages` etc. might work for your purposes.